modules/client/src/tokenVoting/internal/client/methods.ts

```diff
@@ -427,17 +429,14 @@ export class TokenVotingClientMethods extends ClientCore
       }
       return {
         minDuration: parseInt(tokenVotingPlugin.minDuration),
-        minSupport: decodeRatio(
-          parseFloat(
-            tokenVotingPlugin.totalSupportThresholdPct,
-          ),
-          2,
+        supportThreshold: parseFloat(
+          formatEther(tokenVotingPlugin.supportThreshold),
```

Comment:
Why not using our own decoding? We have a separate task to handle this, and I think there's another code section where such change is not made in this PR

Reply:
Our encoder/decoder stopeed making sense when we need to transform 0.5 => 5*10^17 because the precission is not good enough we have like 15 digits so I decide to use `formatEther`and `parseEther` that have the exact behaivour that we want. This was just a test of both this functions to see if I coulñd use them in the other PR, it worked and left it there